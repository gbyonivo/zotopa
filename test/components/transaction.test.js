import React from 'react';
import { shallow } from 'enzyme';
import Transaction from '../../src/components/transaction';


describe('Transaction', () => {
  it('should render correctly', () => {
    const actual = shallow(<Transaction transaction={{ amount: 10, name: 'ivo', email: '3', date: 'd' }}/>);
    expect(actual).toMatchSnapshot();
  });
});