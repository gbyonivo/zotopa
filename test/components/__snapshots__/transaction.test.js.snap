// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transaction should render correctly 1`] = `
<WithContext(TrInner)>
  <WithContext(TdInner)>
    ivo
  </WithContext(TdInner)>
  <WithContext(TdInner)>
    3
  </WithContext(TdInner)>
  <WithContext(TdInner)>
    Â£
    10
  </WithContext(TdInner)>
  <WithContext(TdInner)>
    d
  </WithContext(TdInner)>
</WithContext(TrInner)>
`;
